terminus/tests/vertex_graph_to_roads_converter_test.py

```diff
@@ -104,10 +108,13 @@ def test_get_roads_multiple_independent_segments(self):
         self.converter = VertexGraphToRoadsConverter(
             0.25, [v1, v2, v3, v4, v5])
         roads = self.converter.get_roads()
-        expected_roads = [
-            Street.from_points([Point(0, 0), Point(1, 0), Point(6, 1)]),
-            Street.from_points([Point(2, 5), Point(3, 4)])
-        ]
+        expected_roads = []
```

Comment:
In order to avoid the code duplication in `street.set_width`, could you roll back to the previous definition and add a `set_streets_width()` method that you can apply to the entire collection to use the new width? So for this case the code would be:

```
expected_roads = [
    Street.from_points([Point(0, 0), Point(1, 0), Point(6, 1)]),
    Street.from_points([Point(2, 5), Point(3, 4)])
 ]
set_streets_width(expected_roads)
```